<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveMonitoringPage</class>
 <widget class="QWidget" name="LiveMonitoringPage">
  <property name="styleSheet">
   <string notr="true">
    background-color: #1e293b;
    color: #e2e8f0;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>20</number>
   </property>

   <!-- ==================== STATUS BAR ROW ==================== -->
   <item>
    <widget class="QWidget" name="status_bar">
     <property name="styleSheet">
      <string notr="true">
       background-color: #334155;
       border-radius: 12px;
       padding: 12px;
      </string>
     </property>
     <layout class="QHBoxLayout" name="statusRow">
      <property name="spacing">
       <number>24</number>
      </property>

      <item>
       <widget class="QLabel" name="model_label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Model: —</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QLabel" name="limit_label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Limits: — mm</string>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="statusSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <item>
       <widget class="QLabel" name="value_label">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #60a5fa;</string>
        </property>
        <property name="text">
         <string>Live: — mm</string>
        </property>
       </widget>
      </item>

      <item>
       <spacer name="statusSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>

      <item>
       <widget class="QLabel" name="passfail_label">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Status: —</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- ==================== PLOT CONTAINER ==================== -->
   <item>
    <widget class="QWidget" name="plot_container">
     <property name="styleSheet">
      <string notr="true">
       background-color: #0f172a;
       border: 2px solid #334155;
       border-radius: 16px;
      </string>
     </property>
     <layout class="QVBoxLayout" name="plotLayout">
      <property name="margin">
       <number>12</number>
      </property>
     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>